cmd/minikube/cmd/node_start.go

```diff
@@ -70,6 +70,7 @@ var nodeStartCmd = &cobra.Command{
 		if err != nil {
 			_, err := maybeDeleteAndRetry(*cc, *n, nil, err)
```

Comment:
should we check for the error here as well, to avoid an unnecessary retry?

Reply:
I think this one is okay, since if the user expelicty say delete-on-failure, we should let them do that.